src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig

```diff
@@ -196,8 +196,12 @@
                         </p>
                         <ul>
                             {% for key, value in requirements.testsRequired %}
-                                {% if 'fail' == value and key in requirements.testsErrors|keys %}
-                                    <li>{{ requirements.testsErrors[key] }}</li>
+                                {% if 'fail' == value %}
+                                        {% if key in requirements.testsErrors|keys %}
+                                            <li>{{ requirements.testsErrors[key] }}</li>
+                                        {% else %}
+                                            <li>{{ '%key% (missing description)'|trans({'%key%': key}, 'Admin.Advparameters.Notification') }}</li>
```

Comment:
I'm not sure we should patch this in the template, but ensure the key is in requirements.testsError array, don't you think?